images/capi/ansible/roles/sysprep/tasks/debian.yml

```diff
@@ -75,25 +80,6 @@
     - { path: /var/lib/apt/lists, state: absent, mode: "0755" }
     - { path: /var/lib/apt/lists, state: directory, mode: "0755" }
 
-- name: Disable apt-daily services
-  ansible.builtin.systemd:
-    name: "{{ item }}"
-    state: stopped
-    enabled: false
-  loop:
-    - apt-daily.timer
-    - apt-daily-upgrade.timer
-
-- name: Get installed packages
-  ansible.builtin.package_facts:
-
-- name: Disable unattended upgrades if installed
-  ansible.builtin.systemd:
-    name: unattended-upgrades
-    state: stopped
-    enabled: false
-  when: "'unattended-upgrades' in ansible_facts.packages"
-
```

Comment:
Looks like there are tests that check that these are disabled and fail the build

<details><summary>log snippet</summary>

```
    qemu:         {
    qemu:             "duration": 14848565,
    qemu:             "err": null,
    qemu:             "expected": [
    qemu:                 "false"
    qemu:             ],
    qemu:             "found": [
    qemu:                 "true"
    qemu:             ],
    qemu:             "human": "Expected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "meta": null,
    qemu:             "property": "enabled",
    qemu:             "resource-id": "apt-daily-upgrade.timer",
    qemu:             "resource-type": "Service",
    qemu:             "result": 1,
    qemu:             "successful": false,
    qemu:             "summary-line": "Service: apt-daily-upgrade.timer: enabled:\nExpected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "test-type": 0,
    qemu:             "title": ""
    qemu:         },
    qemu:         {
    qemu:             "duration": 7506652,
    qemu:             "err": null,
    qemu:             "expected": [
    qemu:                 "false"
    qemu:             ],
    qemu:             "found": [
    qemu:                 "true"
    qemu:             ],
    qemu:             "human": "Expected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "meta": null,
    qemu:             "property": "running",
    qemu:             "resource-id": "apt-daily-upgrade.timer",
    qemu:             "resource-type": "Service",
    qemu:             "result": 1,
    qemu:             "successful": false,
    qemu:             "summary-line": "Service: apt-daily-upgrade.timer: running:\nExpected\n    \u003cbool\u003e: true\nto equal\n    \u003cbool\u003e: false",
    qemu:             "test-type": 0,
    qemu:             "title": ""
    qemu:         },
```

</details>

Reply:
Could be that there is openstack specific tests that need to be changed too